packages/runtime/lib/runtime.js

```diff
@@ -1323,6 +1323,17 @@ class Runtime extends EventEmitter {
     return worker
   }
 
+  async #getHealth (worker) {
+    if (features.node.worker.getHeapStatistics) {
+      const { used_heap_size: heapUsed, total_heap_size: heapTotal } = await worker.getHeapStatistics()
+      const { utilization: elu } = worker.performance.eventLoopUtilization()
```

Comment:
This is incorrect. You need to keep track of the last ELU. See https://github.com/platformatic/platformatic/blob/91c05b7fa85700583cdaac93bf4c94ae0cccf7ef/packages/runtime/lib/worker/app.js#L214-L215.